examples/erc1155/tests/abi/mod.rs

```diff
@@ -13,6 +13,9 @@ sol!(
         function safeBatchTransferFrom(address from, address to, uint256[] memory ids, uint256[] memory values, bytes memory data) external;
         function mint(address to, uint256 id, uint256 amount, bytes memory data) external;
         function mintBatch(address to, uint256[] memory ids, uint256[] memory amounts, bytes memory data) external;
+        #[derive(Debug)]
```

Comment:
Why do you need `Debug` here?

Reply:
Probably copy paste mistake, will remove